src/Core/Domain/SqlManagement/QueryHandler/GetSqlRequestSettingsHandler.php

```diff
@@ -54,12 +51,14 @@ public function __construct(ConfigurationInterface $configuration)
     /**
      * {@inheritdoc}
      */
-    public function handle(GetSqlRequestSettings $query)
+    public function handle(GetSqlRequestSettings $query): SqlRequestSettings
     {
         $fileEncodingIntValue = $this->configuration->get(SqlRequestSettings::FILE_ENCODING);
+        $fileSeparatorValue = $this->configuration->get(SqlRequestSettings::FILE_SEPARATOR);
 
         return new SqlRequestSettings(
-            $this->getFileEncoding($fileEncodingIntValue)
+            $this->getFileEncoding($fileEncodingIntValue),
+            $this->getFileSeparator($fileSeparatorValue)
```

Comment:
```suggestion
            $this->getFileSeparator($fileSeparatorValue ?? ';')
```
Same comment it's a bit overkill. But these are just suggestions, not blocking.